content/troubleshooting/net/GettingNetAgentLogs.md

```diff
@@ -68,10 +68,16 @@ Complete the following steps to gather information to send to Contrast.
   * [Stop the .NET agent service](http://127.0.0.1:9000/installation-netusage.html#usage).
   * Enable additional logging. 
      * ** Start > Notepad > Right click > Run as Administrator**
-     * ** File > Open >** *C:\Program Files\Contrast\dotnet\DotnetAgentService.exe.config*
-     * Change `ShouldLogMethodSignatures` to `true`.
-     * Uncomment the entry for `LogLevel`. 
-     * Check that the specified `LogLevel` value is `trace`.
+     * ** File > Open >** *C:\ProgramData\Contrast\dotnet\contrast_security.yaml*
```

Comment:
Do we need to edit this as admin anymore? I believe `C:\ProgramData\Contrast\dotnet` inherits from `C:\` which doesn't require admin, unless the installer modifies something.

Reply:
removed the 'start as admin' text.